src/backend/data/models/gc-audit.js

```diff
@@ -0,0 +1,28 @@
+export default function(sequelize, DataTypes) {
+  let GCAudit = sequelize.define('GCAudit', {
+    id: {
+      type: DataTypes.INTEGER,
+      field: 'id',
+      primaryKey: true
+    },
+    class: {
+      type: DataTypes.STRING
+    },
+    method: {
+      type: DataTypes.STRING
+    },
+    params: {
+      type: DataTypes.STRING
+    },
+    error: {
+      type: DataTypes.STRING
+    }
+  }, {
+    underscored: true,
```

Comment:
I've just standardized on adding modified_dt and create_dt everywhere.  See the other models' createdAt and updatedAt methods.